src/components/CommentsCard.jsx

```diff
@@ -0,0 +1,39 @@
+import { getComments } from "../utils/api";
+import { useEffect, useState } from "react";
+
+const CommentsCard = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    getComments(article_id)
+      .then((res) => {
```

Comment:
We should implement a loading state so the users know the comments are still loading